ci/templates/maven-test-project/check.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
- Merge https://youtrack.jetbrains.com/issue/CMP-7738/Compose-fails-to-sync-in-Bazel fix
- Merge this PR
- Modify the check on CI

Reply:
https://jetbrains.team/p/ui/repositories/compose-teamcity-config/revision/05477a4d7da9d7f980cb0063485ec09dc490a34f

Reply:
I reverted the check. There some warnings on CI:
https://teamcity.jetbrains.com/buildConfiguration/JetBrainsPublicProjects_Compose_Dev/5048030?buildTab=dependencies&mode=list&state=failed&type=snapshot&showLog=5048026_13096_379&logFilter=debug&logView=flowAware#5048026

Reply:
https://youtrack.jetbrains.com/issue/CMP-7816/Space-maven-repo-doesnt-contain-checksums-for-some-versions